### ISSUE TITLE
Get Airtable access for new base: Customs pain

### ISSUE THREAD

**Issue by v1ctorio** [hackclub/infra#98]
Desired Airtable base name: customspain
Reason for needing access: I would like to use airtable instead of Slack's list in my new version of Johnny for the customs pain centre, so we can have a crowdsourced database of customs experiences.

**Comment by grymmy** [comment]
I don't think this necessitates a new airtable DB - something as simple as [Google forms + Google sheets](https://support.google.com/docs/answer/6281888?hl=en&co=GENIE.Platform%3DDesktop#zippy=%2Ccreate-a-form-in-google-sheets) would suffice...
